docs/book/intro.md

```diff
@@ -1,7 +1,7 @@
 # Introduction to zend-console
 
-## Deprecated
-Both the zend-console an the [zend-mvc-console](https://docs.zendframework.com/zend-mvc-console/) components will likely not be maintained long-term, as there are more complete implementations available elsewhere. We strongly urge developers to start migrating their console tooling to use other libraries, such as [symfony/console](https://github.com/symfony/console).
+> ### Deprecated
+> Both the zend-console an the [zend-mvc-console](https://docs.zendframework.com/zend-mvc-console/) components will likely not be maintained long-term, as there are more complete implementations available elsewhere. We strongly urge developers to start migrating their console tooling to use other libraries, such as [symfony/console](https://github.com/symfony/console).
```

Comment:
Please add an empty line after the headline. (See at my example.) Thanks!

Reply:
Gee. Sorry. ;-)